lib/i18n.ts

```diff
@@ -0,0 +1,27 @@
+import { match } from "@formatjs/intl-localematcher";
+import type e from "express";
+import Negotiator from "negotiator";
+
+declare global {
+  namespace Express {
+    interface Request {
+      i18n: {
+        language: string;
+      };
+    }
+  }
+}
+
+export default {
+  accept() {
+    return async (req: e.Request, res: e.Response, next: e.NextFunction) => {
+      const n = new Negotiator(req);
```

Comment:
what is a language negotiator?

Reply:
it's a library for doing http content negotation; https://github.com/jshttp/negotiator

Rather than explicitly including a "language" in every query, the graphql stuff just relies on standard http headers when fetching dynamic text. The console sets this header to the worker (instance)'s preferred language when it makes graphql calls.